packages/web-components/fast-foundation/src/listbox-option/listbox-option.ts

```diff
@@ -217,7 +217,7 @@ export class ListboxOption extends FoundationElement {
         Observable.notify(this, "value");
     }
 
-    public get value(): string {
+    public get value(): string | unknown {
```

Comment:
@codef0rmerz you'll need to run `yarn change` for this to generate a change file for this patch.